jplag.frontend.kotlin/src/main/java/de/jplag/kotlin/Language.java

```diff
@@ -2,15 +2,18 @@
 
 import java.io.File;
 
+import org.kohsuke.MetaInfServices;
+
 import de.jplag.TokenList;
 
 /**
  * This represents the Kotlin language as a language supported by JPlag.
  */
+@MetaInfServices(de.jplag.Language.class)
 public class Language implements de.jplag.Language {
 
     private static final String NAME = "Kotlin Parser";
-    private static final String SHORT_NAME = "Kotlin";
+    public static final String SHORT_NAME = "kotlin";
```

Comment:
Maybe we rather rename this to `IDENTIFIER`?

Reply:
I agree; we could do that. I simply used the SHORT_NAME everywhere. But IDENTIFIER would be better (we could also rename the `getShortName()`)

What do you think @tsaglam

Reply:
Identifier sounds good. The getter has the benefit that devs of new frontends are forced to implement such a method due to the contract of the Language interface. However, we can also remove it if the constant is directly accessed anyways.